Automattic/wp-calypso
# Adds new package @automattic/calypso-doctor

#### Changes proposed in this Pull Request

Adds a new package `@automattic/calypso-doctor`. This package is a tool that performs some tests on the developer's system to verify they have the best settings to develop Calypso smoothly.

It is designed to be modular, adding new tests should be easy. In the future it can be expanded to automatically fix the problems, validate other sets of tests (for example, VSCode settings)...

For now it runs on `postinstall` on the main project, but it doesn't fail the installation in any way. Is up to the developer to manually apply the fixes.

Example of a run:

![image](https://user-images.githubusercontent.com/975703/88055188-db031c80-cb5e-11ea-8318-80cecc536c7b.png)

After applying the suggested fixes:

![image](https://user-images.githubusercontent.com/975703/88055276-fc640880-cb5e-11ea-98fb-2b77a9db03eb.png)

#### Testing instructions

Checkout the branch, run `yarn` and see what errors you get. Try to apply the suggested fixes and see if the errors go away.
